google/cloud/internal/async_read_write_stream_auth.h

```diff
@@ -128,6 +128,20 @@ class AsyncStreamingReadWriteRpcAuth
   std::shared_ptr<SharedState> const state_;
 };
 
+template <typename Request, typename Response>
+std::unique_ptr<AsyncStreamingReadWriteRpc<Request, Response>>
+MakeAuthorizedStreamingReadWriteRpc(
+    CompletionQueue const& cq, std::unique_ptr<grpc::ClientContext> context,
+    std::shared_ptr<google::cloud::internal::GrpcAuthenticationStrategy> auth,
+    PrepareAsyncReadWriteRpc<Request, Response> async_call) {
+  return absl::make_unique<AsyncStreamingReadWriteRpcAuth<Request, Response>>(
+      std::move(context), std::move(auth),
+      [=](std::unique_ptr<grpc::ClientContext> ctx) {
```

Comment:
Note that `async_call` is a `absl::FunctionRef`:

https://github.com/googleapis/google-cloud-cpp/blob/6b4e66f77f88246fe39a244035ff0e1bf9b261f5/google/cloud/internal/async_read_write_stream_impl.h#L166-L169

Which should not be deep copied:

https://github.com/abseil/abseil-cpp/blob/f523d0dd69806d8057a898bb2595910558156aaa/absl/functional/function_ref.h#L36-L39

Reply:
Ack, reverted this change.